easybuild/easyblocks/g/gromacs.py

```diff
@@ -90,18 +92,18 @@ def configure_step(self):
                     libs = os.getenv('LIB%s' % libname)
                     self.cfg.update('configopts', '-DGMX_%s_USER="-L%s %s"' % (libname, lib_dir, libs))
 
+            # set regression test path
+            prefix = 'regressiontests'
+            if any([src['name'].startswith(prefix) for src in self.src]):
+                self.cfg.update('configopts', "-DREGRESSIONTEST_PATH='%%(builddir)s/%s-%%(version)s' " % prefix)
+
+        if LooseVersion(self.version) < LooseVersion('5.0'):
```

Comment:
right, no more GSL in 5.x series: http://www.gromacs.org/About_Gromacs/Release_Notes/Versions_5.0.x